pkg/loop/internal/pb/capabilities_registry.proto

```diff
@@ -0,0 +1,107 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/pb";
+
+package loop;
+
+import "values/pb/values.proto";
+import "google/protobuf/empty.proto";
+
+service CapabilitiesRegistry {
+  rpc Get (GetRequest) returns (GetReply) {}
+}
+
+enum ExecuteAPIType {
+  EXECUTE_API_TYPE_UNKNOWN = 0;
+  EXECUTE_API_TYPE_TRIGGER = 1;
+  EXECUTE_API_TYPE_CALLBACK = 2;
+}
+
+// Get has arguments for [github.com/smartcontractkit/chainlink-common/pkg/types.CapabilitiesRegistry.Get].
+message GetRequest {
+  string id = 1;
+}
+
+message GetReply {
+  uint32 capabilityID = 1;
+  ExecuteAPIType type = 2;
+}
+
+service BaseCapability {
+  rpc Info (google.protobuf.Empty) returns (CapabilityInfoReply) {}
+}
+
+enum CapabilityType {
+  CAPABILITY_TYPE_UNKNOWN = 0;
+  CAPABILITY_TYPE_TRIGGER = 1;
+  CAPABILITY_TYPE_ACTION = 2;
+  CAPABILITY_TYPE_CONSENSUS = 3;
+  CAPABILITY_TYPE_TARGET = 4;
+}
+
+message CapabilityInfoReply {
+  string id = 1; 
+  CapabilityType capability_type = 2;
+  string Description = 3;
```

Comment:
you have an odd admixture of upper case, lower case, snake case and camel case in the field defintions through this file. https://protobuf.dev/programming-guides/style/#message-field-names

Reply:
ugh, yes let me correct this